### PR TITLE
docs(changelog): release 2.1.3

### PR DESCRIPTION
**Pull request by lengau**
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?

-----

- Also cherry-picks the docs fixes from #872 
- Includes a fix to make the docs linters pass

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+
+- Fixed an issue where the ``poetry`` plugin would still try to install poetry
+  from the package repositories with a ``poetry-deps`` dependency declared.
+
+Documentation:
+
+- Added some missing references in the
+  :doc:`Poetry plugin</common/craft-parts/reference/plugins/poetry_plugin>` and
+  :doc:`Python plugin</common/craft-parts/reference/plugins/python_plugin>` pages.
+- Fixed a broken link in the :doc:`Tutorial examples</tutorials/examples>`.
```
Comment (medubelko) [inline docs/changelog.rst:18]:
```suggestion
- Fix an issue where the ``poetry`` plugin would still try to install poetry
  from the package repositories with a ``poetry-deps`` dependency declared.

Documentation:

- Add some missing references in the
  :doc:`Poetry plugin</common/craft-parts/reference/plugins/poetry_plugin>` and
  :doc:`Python plugin</common/craft-parts/reference/plugins/python_plugin>` pages.
- Fix a broken link in the :doc:`Tutorial examples</tutorials/examples>`.
```

File: docs/changelog.rst [hunk 2]
```diff
@@ -2,6 +2,23 @@
 Changelog
 *********
 
+2.1.3 (2024-11-20)
+------------------
+
+Bug fixes:
+
+- Fixed an issue where the ``poetry`` plugin would still try to install poetry
+  from the package repositories with a ``poetry-deps`` dependency declared.
```
Comment (medubelko) [inline docs/changelog.rst:11]: Can we simplify the grammar with:

```suggestion
  from the package repositories when ``poetry-deps`` was declared as a
  dependency
```

**Review by medubelko — CHANGES_REQUESTED** [review]
Thanks for prepping this fix! :pray: 

Two minor suggestions, one purely for consistency.

File: docs/changelog.rst (hunk 1 above)
Comment (lengau) [reply docs/changelog.rst]: Done!

File: docs/changelog.rst (hunk 2 above)
Comment (lengau) [reply docs/changelog.rst]: Done!

**Review by medubelko — APPROVED** [review]
LGTM!

**Review by mr-cal — APPROVED** [review]

**Pull request merged**